src/main/resources/application.properties

```diff
@@ -82,6 +82,7 @@ amppd.refreshResultsStatusMinutes = 10
 amppd.refreshResultsTableMinutes = 300
 amppd.refreshWorkflowResultsStatusCron = 0 0/10 6-18 ? * MON-FRI
 amppd.refreshWorkflowResultsAllCron = 0 0 1 ? * MON-FRI
+amppd.environment = "Test"
 
```

Comment:
Lets put "Dev" in the default property, as for other properties. Will need to add this to test/staging and bamboo.

Reply:
For Bamoo, we can call it "CI-Bamboo".